echopype/visualize/api.py

```diff
@@ -80,7 +125,7 @@ def create_echogram(
                 ek_encode_mode=range_kwargs.get('encode_mode', 'complex'),
             )
             range_in_meter.attrs = range_attrs
-            if 'water_level' in data.platform:
+            if water_level is True and 'water_level' in data.platform:
                 # Adds water level to range if it exists
                 range_in_meter = range_in_meter + data.platform.water_level
```

Comment:
Right now the `water_level` is int, float, or xr.DataArray cases do not work with when `get_range=True` and data is an `EchoData` object, since this only allows pulling the water_level values from the Platform group. 

It seem that the block you have from below would work for both the data is Echodata and data is Sv cases and can largely be repurposed?

Reply:
Ah I see, so you'd want user to be able to pass in their on water level also on the case with EchoData object?

Reply:
Yes! since there is no guarantee that the water_level saved in the data is correct, and it is often not populated.